sycl/CMakeLists.txt

```diff
@@ -415,11 +415,6 @@ option(SYCL_INCLUDE_TESTS
   "Generate build targets for the SYCL unit tests."
   ${LLVM_INCLUDE_TESTS})
 
-# Ensure that HIP platform is uppercase, to match buildbot's output.
```

Comment:
Can we remove support outside of an ABI breaking period?

Reply:
yes, exactly. The change here is test-only.